### PR TITLE
feat: currency & safeLong

### PR DESCRIPTION
**Pull request by coolme200**

### CODE REVIEW

**Comment by coolme200** [comment]
@dead-horse @fengmk2 see see ?

File: index.js [hunk 1]
```diff
@@ -85,6 +91,13 @@ function string(val) {
   return String(val);
 }
 
+function safeLong(val) {
+  if (typeof val !== 'string') {
+    console.warn('Long value: %d is a Number, Use String type will be more safe!', val);
```
Comment (fengmk2) [inline index.js]: Only show on dev env. `process.env.NODE_ENV !== 'production'`

Comment (fengmk2) [inline index.js]: Or just use http://npm.taobao.org/package/depd

**Comment by fengmk2** [comment]
+1

**Comment by fengmk2** [comment]
2.2.0

**Comment by fengmk2** [comment]
```
npm owner ls
dead_horse <dead_horse@qq.com>
coolme200 <tangyao@alibaba-inc.com>
```

@coolme200 我没有发布权限。。。

**Comment by coolme200** [comment]
`js-to-java@2.2.0`

**Pull request merged**